src/types/UiaTextRangeBase.hpp

```diff
@@ -218,6 +218,9 @@ namespace Microsoft::Console::Types
 
         IRawElementProviderSimple* _pProvider;
 
+        // TODO CARLOS: import this value on UTR construction
+        std::wstring _wordDelimiters = L" ";
+

```

Comment:
Whoops! Just noticed this `TODO CARLOS`. @ZoeyR Changing this variable would be how we accomplish defining your own delimiter runs. I could make this defined in the `ScreenInfoUiaProvider` then pass it down to the `UiaTextRange`. After some thought, I'm just not sure how useful defining your own delimiters would be, though?

Also, if you define your own delimiters (i.e.: "." and "/"), the current behavior _for selection_ is to stop at a delimiter class change. There are 3 delimiter classes:
1. whitespace and control characters
2. defined delimiters
3. other chars
Check out the PR description [here](https://github.com/microsoft/terminal/pull/2511).

Is that a wanted behavior or do we want to just have two classes (merge class 1 and 2)?

Reply:
I'm not sure we should affect selection behavior via this code. These delimiters should mostly be for navigating the text via "word" breakpoints. It would be useful to have things like `\` and `/` as delimiters so that path navigation would be easier, this is what NVDA does for instance.

Reply:
The plan wouldn't be to affect the selection behavior, but rather reuse some existing code. The concept of understanding what a word is is pretty similar between selection and accessibility so I figured I'd reuse them.

As for importing your own word delimiters, I just added them in as an optional parameter, so they'll be in the upcoming commit to this PR. 😊

Reply:
Actually, they're added as an optional param to the UiaTextRange. I'm guessing you'd need that to be an option at the ScreenInfoUiaProvider level. I'll add that in in the next one.

Reply:
Created a new PR targeting this one to enable this feature. It's a weird change (in the context of this one) and this PR is already pretty big. Check out #3805